.gitignore

```diff
@@ -3,3 +3,5 @@
 /composer.lock
 /phpunit.xml
 /vendor
+/.phpunit.result.cache
+/.idea
```

Comment:
this belongs in your global gitignore file

see https://gist.github.com/subfuzion/db7f57fff2fb6998a16c for more

Reply:
Thx for gist) Useless options removed!